README.md

```diff
@@ -8,25 +8,24 @@ The docker images are available on [Amazon Corretto Official Images](https://hub
 
 To use Amazon Corretto Official Images from Docker hub run
 ```
-docker pull amazoncorretto:11
+docker pull amazoncorretto:latest
 ```
 
 The docker images are also available on **Amazon ECR**.
 
-To use the docker images from Amazon ECR, you would need to authenticate with the ECR registry (id: 489478819445) with the
-help of instruction from [here](https://aws.amazon.com/blogs/compute/authenticating-amazon-ecr-repositories-for-docker-cli-with-credential-helper/).
-Once authenticated, Amazon Corretto docker images can be pulled using command
+To use the docker images from Amazon ECR, avilable images can be found in the public ECR repository. 
```

Comment:
This doesn't quite flow right to me.. Maybe something like this:

To get Corretto docker images from Amazon ECR please see [Amazon Corretto's ECR Public Gallery](https://gallery.ecr.aws/amazoncorretto/amazoncorretto) as well as the [Docker Official Images ECR Public Gallery](https://gallery.ecr.aws/docker/library/amazoncorretto) 


typo: available